src/plugins/discover/public/application/view_components/utils/filter_columns.ts

```diff
@@ -17,11 +18,16 @@ import { IndexPattern } from '../../../opensearch_dashboards_services';
 export function filterColumns(
```

Comment:
Need to add @param for the new `modifyColumn` above. Also need to modify the above function description.

Reply:
Added